File: binarysearch.io/280_minimum_difference.cpp
int solve(vector<int>& lst1, vector<int>& lst2) {
    sort(lst1.begin(),lst1.end());
    sort(lst2.begin(),lst2.end());
    int i , j;
    i = j = 0;
    int diff = INT_MAX;
    while(i < lst1.size() && j < lst2.size()){
        diff = min(abs(lst1[i]-lst2[j]),diff);
        if(lst1[i] < lst2[j]) i++;
        else if(lst1[i] > lst2[j]) j++;
        else return diff;
    }
    return diff;
}

File: binarysearch.io/280_minimum_difference.py
class Solution:
    def solve(self, lst1, lst2):
        lst1.sort()
        lst2.sort()
        n = len(lst1)-1
        m = len(lst2)-1
        
        i = 0 
        j = 0
        ans = float('inf')
        while i <= n and j <= m:
            ans = min(abs(lst1[i]-lst2[j]),ans)
            if lst1[i] < lst2[j]:
                i += 1 
            elif lst1[i] > lst2[j]:
                j += 1 
            else:
                return ans 
        return ans 
            
        
        